infoblox-ipam.go

```diff
@@ -72,13 +72,26 @@ func (ibDrv *InfobloxDriver) GetAddress(netviewName string, cidr string, ipAddr
 	return fixedAddr, err
 }
 
-func (ibDrv *InfobloxDriver) RequestAddress(netviewName string, cidr string, ipAddr string, macAddr string, vmID string) (string, error) {
+func (ibDrv *InfobloxDriver) RequestAddress(netviewName string, cidr string, ipAddr string, macAddr string, gw net.IP, vmID string) (string, error) {
 	var fixedAddr *ibclient.FixedAddress
-
+	gateway := gw.String()
 	if netviewName == "" {
 		netviewName = ibDrv.DefaultNetworkView
 	}
 
+	if gateway != "" {
```

Comment:
Please add description for the logic and also mention that as cni donot provide gateway creation call we are doing internally.

Reply:
ok will do that